src/views/MakingARequest/index.test.tsx

```diff
@@ -44,7 +48,9 @@ describe('The making a request page', () => {
       </MemoryRouter>
     );
 
-    expect(screen.getByTestId('making-a-system-request')).toBeInTheDocument();
+    await waitFor(() => {
```

Comment:
Was this why unhandled rejections were making it through? Is it possible to `await expect` here instead, or is `waitFor` critical?

Reply:
Yes this is why.  The oktaAuth.getUser() returns a promise that then changes the state.  The waitfor I believe is necessary to handle the resolved promise and rerender.